src/custom-elements/src/components/tabs/tabs.tsx

```diff
@@ -317,9 +342,10 @@ export class Tabs {
   };
 
   handlerClickTab(e: Event, tabData: TabTrigger, slidingBorderNewPosition?: HTMLElement) {
-    if (this.isTabAnchorLink(tabData)) {
-      e.preventDefault();
-    }
+    if (!this.isTabInternalLink(tabData)) return;
```

Comment:
This will stop the event from being emitted for external links. I'm not sure if that's the correct behaviour.

Reply:
To test, I used the tab configuration used in the documentation for tabbed navigation:

https://lib.lumen.com/chi/6.3.0/components/tabs/?theme=Portal#tabbed-navigation-bar

Reply:
When testing in the panels example, the active class is not set for external links:

![image](https://github.com/user-attachments/assets/46aa85cf-06f4-40c7-ae60-8ca34fb37110)

the blue color of tab-3 disappears on hovering the page, but tab1 is still the active one. It may make sense for external tabs not to have the active class, since it's opening a new tab or doing a redirection.